microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -698,15 +698,15 @@ Windows Information Protection (WIP) helps prevent accidental leaking of corpora
    Windows Information Protection
 :::image-end:::
 
-If WIP is turned on for your version of Microsoft Edge, the browser collects event logs and sends them to your organization.  If WIP is turned on, you are not able to opt out of the data collection.  WIP is only available on releases of Windows 10 from August 2016 or later.  For more information about the event logs captured by WIP, navigate to [How to collect Windows Information Protection (WIP) audit event logs][WindowsSecurityInformationProtectionCollectAuditEventLogs].
+If WIP is turned on for your version of Microsoft Edge, the browser collects event logs and sends them to your organization.  If WIP is turned on, you are not able to opt out of the data collection.  WIP is only available on releases of Windows 10 from August 2016 or later.  For more information about the event logs captured by WIP, navigate to [How to collect Windows Information Protection (WIP) audit event logs](/windows/security/information-protection/windows-information-protection/collect-wip-audit-event-logs).
 
 
 <!-- ====================================================================== -->
 ## Thank you!
 
-Microsoft Edge is made possible by the [Chromium][ChromiumMain] open-source project and other open-source software.  To view all of the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper][GoogleChromePrivacyWhitepaper] was used as a source for gathering related information about the Chromium open source project.
+Microsoft Edge is made possible by the [Chromium](https://www.chromium.org) open-source project and other open-source software.  To view all of the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper](https://www.google.com/chrome/privacy/whitepaper.html) was used as a source for gathering related information about the Chromium open source project.
 
-Microsoft Edge is made possible by the [Chromium][ChromiumMain] open-source project and other open-source software.  To view all of the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper][GoogleChromePrivacyWhitepaper] was used as a source for gathering related information about the Chromium open-source project.
+Microsoft Edge is made possible by the [Chromium](https://www.chromium.org) open-source project and other open-source software.  To view all of the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper](https://www.google.com/chrome/privacy/whitepaper.html) was used as a source for gathering related information about the Chromium open-source project.
```

Comment:
@captainbrosset @dalenaMSFT @vchapel - here is the dup paragraph.  It got uncommented and revised in a previous PR.
```suggestion
```

Reply:
Public rendered article section: https://docs.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#thank-you

Reply:
Public article's section: https://docs.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#thank-you

Out of context, it might look like my Suggestion reverts simple links to complicated link tagging, but the version of the paragraph that remains above it has the parentheses fix.

Reply:
Ok, so I will leave it as is in this PR, since 1480 will take care of it. I prefer to keep this PR only focused on changing the links, so as to minimize merge conflicts.